miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultContactRoleService.java

```diff
@@ -86,7 +86,7 @@ public ValidationResult validateDeletion(ContactRole object) throws IOException
     ValidationResult result = new ValidationResult();
     long usage = contactRoleDao.getUsage(object);
     if (usage > 0L) {
-      result.addError(ValidationError.forDeletionUsage(object, usage, Pluralizer.projects(usage)));
+      result.addError(ValidationError.forDeletionUsage(object, usage, Pluralizer.contacts(usage)));
```

Comment:
Change to `"project" + Pluralizer.contacts(usage)` - this is specifically a count of project-contacts. The same contact will be counted multiple times if associated with the same role in multiple projects